src/test/java/utilities/Util.java

```diff
@@ -14,5 +18,17 @@ public static void wait(int seconds) {
 			e.printStackTrace();
 		}
 	}
+	
+	/*
+	 * switch to iFrame
+	 * 
+	 * @Author: Andy Zeng
+	 */
+	public static void iFrame(WebDriver driver, String frameElement) {
```

Comment:
1, bad parameter naming, `frameElement` is actually used as an xpath
2, Don't pass in an xpath, make it a `By` object to keep it more reusable
3, Format the file